docs/en-US/component/tour.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Tour
+lang: en-US
+---
+
+# Tour
+
+A popup component for guiding users through a product. Use when you want to guide users through a product.
+
+## Basic usage
+
+The most basic usage
+
+:::demo
+
+tour/basic
+
+:::
+
+## Non modal
+
+Use `:mask="false"` to make Tour non-modal. At the meantime it is recommended to use with `type="primary"` to emphasize the guide itself.
+
+:::demo
+
+tour/non-modal
+
+:::
+
+## Placement
+
+Change the placement of the guide relative to the target, there are 12 placements available. When `target` is empty the guide will show in the center.
+
+:::demo
+
+tour/placement
+
+:::
+
+## Custom mask style
+
+Custom mask style.
+
+:::demo
+
+tour/mask
+
+:::
+
+## Custom indicator
+
+Custom indicator.
+
+:::demo
+
+tour/indicator
+
+:::
+
+
+## Tour API
+
+:::tip
+tour-step component configuration with the same name has higher priority
+:::
+
+### Tour Attributes
+
+| Property | Description | Type | Default |
+| --- | --- | --- | --- |
+| show-arrow | whether to show the arrow | `boolean` | true |
+| placement | position of the guide card relative to the target element | ^[enum]`'top' \| 'top-start' \| 'top-end' \| 'bottom' \| 'bottom-start' \| 'bottom-end' \| 'left' \| 'left-start' \| 'left-end' \| 'right' \| 'right-start' \| 'right-end'` | `bottom` |
+| content-style | custom style for content | `CSSProperties` | - |
+| mask | whether to enable masking, change mask style and fill color by pass custom props | `boolean` \| ^[Object]`{ style?: CSSProperties; color?: string; }` | `true` |
+| type | type, affects the background color and text color | `default` \| `primary` | `default` |
+| model-value / v-model | open tour | `boolean` | - |
+| current / v-model:current | what is the current step | `number` | - |
+| scroll-into-view-options | support pass custom scrollIntoView options | `boolean` \| `ScrollIntoViewOptions` | ^[Object]`{ block: 'center' }` |
+| z-index | Tour's zIndex | `number` | `2001` |
+| show-close | whether to show a close button | `boolean` | `true` |
+| close-icon | custom close icon, default is Close | `string` \| `Component` | - |
+| close-on-press-escape | whether the Dialog can be closed by pressing ESC | `boolean` | `true` |
+| target-area-clickable | whether the target element can be clickable, when using mask | `boolean` | `false` |
```

Comment:
```suggestion
| close-on-press-escape | whether the Dialog can be closed by pressing ESC | `boolean` | `false` |
| target-area-clickable | whether the target element can be clickable, when using mask | `boolean` | `true` |
```
I think change the default value would be better.

Reply:
I think it's better to keep the default value of `target-area-clickable` to "false".

Reply:
> I think it's better to keep the default value of `target-area-clickable` to "false".

@btea  @Fuphoenixes  @ryuhangyeong  What do you think?

Reply:
I think the default value of ` close-on-press-escape` should be set to `true`, consistent with other components, such as `dialog`.

It is better to set the default value of `target-area-clickable` to `true`. Referring to other component libraries, there seems to be no relevant configuration, but the targets are all clickable.

Reply:
OK, I changed it